eo-phi-normalizer/src/Language/EO/Phi/Metrics/Collect.hs

```diff
@@ -68,7 +68,7 @@ count x = length . filter x
 -- 1
 countDataless :: (Num a) => [Binding] -> a
 countDataless bindings =
-  let countDeltas = count (\case DeltaBinding _ -> True; _ -> False)
+  let countDeltas = count (\case DeltaBinding _ -> True; DeltaEmptyBinding -> True; _ -> False)
```

Comment:
I don't think this is correct. I think, having any empty binding means the object is dataless.

Reply:
Seems that, according to https://github.com/yegor256/phi-paper/issues/15#issuecomment-1982769768, it was correct, right?